src/spikeinterface/core/basesorting.py

```diff
@@ -347,7 +412,7 @@ def get_all_spike_trains(self, outputs="unit_id"):
             spikes.append((spike_times, spike_labels))
         return spikes
```

Comment:
I think that `to_spike_vector` can be confusing because we have the sorter methods `to_numpy_sorting`, `to_shared_memory_sorting` and `to_dict` that transform the sorter and output a different representation of the sorter.

`to_spike_vector` though just builds a vector representation of the spikes. I think that something like `build_spike_vector` would be easier to discover when someone is exploring the sorter with auto-completion and will help disambiguate the semantic convention that we have in the `to_something` methods.

Reply:
good idea. @alejoe91 are you agree with this naming proposal ?
This could break some old code.

Reply:
yes, sounds good to me

Reply:
`get_spike_vector()`

Reply:
Again, let's keep it for now and change it in a future PR

Reply:
If Sam is fine with `get_spike_vector` I am as well. I think that's an improvement over the existing one that we can release with this big change. If you don't have any reason against I suggest we change it now.